.gitignore

```diff
@@ -131,9 +131,12 @@ dmypy.json
 # PyCharm
 .idea/
 
-# Project Specific
-romfilepath.txt
-
 # Cython
 *.c
 *.html
+
+
+# Project Specific
+romfilepath.txt
+src/test_rom_patcher.py
+src/test_data.json
```

Comment:
Why are these in the gitignore? tests sound like they should be tracked.

Reply:
These are just super lightweight things I use for quickly prototyping things. The py is like a one-liner so I don't forget how to run the program, and I don't use it rn. The test_data is completely incidental and I constantly replace and rewrite it during testing.

Reply:
Cant you put them in a local git ignore instead then?

Reply:
https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally